compiler/src/dotty/tools/dotc/typer/Dynamic.scala

```diff
@@ -215,18 +216,21 @@ trait Dynamic {
         }
 
         if (isDependentMethod(tpe))
-          fail(name, i"has a method type with inter-parameter dependencies")
+          fail(i"has a method type with inter-parameter dependencies")
         else {
-          def ctags = tpe.paramInfoss.flatten.map(pt =>
-            implicitArgTree(defn.ClassTagClass.typeRef.appliedTo(pt.widenDealias :: Nil), fun.span.endPos))
-          structuralCall(nme.applyDynamic, ctags).cast(tpe.finalResultType)
+          def classOfs =
+            if tpe.paramInfoss.exists(_.exists(!TypeErasure.hasStableErasure(_))) then
```

Comment:
```suggestion
            if tpe.paramInfoss.nestedExists(!TypeErasure.hasStableErasure(_)) then
```

Reply:
Thanks. I did not know about that method. Fixed.